api/src/main/java/org/ehrbase/api/service/FolderService.java

```diff
@@ -79,9 +79,8 @@
      * Updates a target folder entry identified by the given folderId with new
      * content. The content string will be serialized from the given source
      * format.
-     * TODO: Copy from CompositionService. Must be designed for folder
      *
-     * @param folderId - Id of the target folder
+     * @param folderId - Full version_uid for folder including system id and version
```

Comment:
@axelsiebert the correct type would be OBJECT_VERSION_ID https://specifications.openehr.org/releases/BASE/latest/base_types.html#_object_version_id_class